.github/actions/secure-builder-checkout/action.yaml

```diff
@@ -0,0 +1,33 @@
+name: "secure-builder-checkout"
+description: "Checkout the builder repository"
+
+inputs:
+  repository:
+    description: "The repository to check out."
+    required: true
+  ref:
+    description: "A fully formed ref (refs/...) or SHA to checkout."
```

Comment:
If we pass this directly to `actions/checkout` it doesn't need to be fully formed. It could just be a tag or branch name.

Reply:
Thanks. I updated the description.